mlflow/go/pkg/config/config_test.go

```diff
@@ -0,0 +1,44 @@
+package config_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/mlflow/mlflow/mlflow/go/pkg/config"
+)
+
+func TestValidDuration(t *testing.T) {
+	t.Parallel()
+
+	samples := []string{
+		"1000",
+		`"1s"`,
+		`"2h45m"`,
+	}
```

Comment:
we should also check that the expected duration got produced, not just the lack of error